CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 
 ### Added
 
+- Added: `Faker.Cannabis` [[@nurugger07][]]
```

Comment:
Since you're first time contributor, you should include your name at the end of this file.

Reply:
@vbrazo done :)